lib/converters/circuit-to-3d.ts

```diff
@@ -187,6 +193,77 @@ export async function convertCircuitJsonTo3D(
     }
 
     boxes.push(boardBox)
+  } else if (drawFauxBoard) {
+    // Create synthetic faux board when no real board exists
+    const pcbComponents = db.pcb_component?.list?.() ?? []
+
+    if (pcbComponents.length > 0) {
+      // Calculate bounds from component positions with 10% padding
+      let minX = Infinity
+      let minY = Infinity
+      let maxX = -Infinity
+      let maxY = -Infinity
+
+      for (const component of pcbComponents) {
+        const halfWidth = component.width / 2
+        const halfHeight = component.height / 2
+
+        minX = Math.min(minX, component.center.x - halfWidth)
+        minY = Math.min(minY, component.center.y - halfHeight)
+        maxX = Math.max(maxX, component.center.x + halfWidth)
+        maxY = Math.max(maxY, component.center.y + halfHeight)
+      }
```

Comment:
use a function from circuit-json-util, dont re-implement bounds calculation for pcb components for the 1000th time